pkgs/applications/kde/default.nix

```diff
@@ -96,7 +96,7 @@ let
       incidenceeditor = callPackage ./incidenceeditor.nix {};
       itinerary = callPackage ./itinerary.nix {};
       juk = callPackage ./juk.nix {};
-      k3b = callPackage ./k3b.nix {};
+      k3b = if config.allowAliases then (callPackage ./k3b.nix {}) else (throw "libsForQt5.k3b no longer builds, use kdePackages.k3b instead");
```

Comment:
Wouldn't that just call the _old_ k3b package if `allowAliases`?

Reply:
Yes it would. So you suggest it should point to the new Qt 6 build instead?

Reply:
Yes, with a warning, and if `allowAliases == false`, it should just not exist at all.